eng/pipelines/aggregate-reports.yml

```diff
@@ -52,13 +50,16 @@ jobs:
       displayName: Generate Interdependency Report Data
 
     - pwsh: |
-        copy -r target/staging $(Build.ArtifactStagingDirectory)
-        copy target/dependency-whitelist.json (Join-Path $(Build.ArtifactStagingDirectory) "staging")
-        copy eng/code-quality-reports/src/main/resources/index.html $(Build.ArtifactStagingDirectory)
-        copy output/pom.client.html (Join-Path $(Build.ArtifactStagingDirectory) "staging")
-        copy eng/common/InterdependencyGraph.html (Join-Path $(Build.ArtifactStagingDirectory) "staging")
-        copy eng/common/dependency-whitelist.html (Join-Path $(Build.ArtifactStagingDirectory) "staging")
-        copy data.js (Join-Path $(Build.ArtifactStagingDirectory) "staging")
+        Copy-Item eng/jacoco-test-coverage/target/staging $(Build.ArtifactStagingDirectory) -Recurse
+        Copy-Item eng/jacoco-test-coverage/target/dependency-whitelist.json $(Build.ArtifactStagingDirectory)\staging
+        Copy-Item eng/code-quality-reports/src/main/resources/index.html $(Build.ArtifactStagingDirectory)
+        
+        # What is this?
+        # copy output/pom.client.html (Join-Path $(Build.ArtifactStagingDirectory) "staging")
```

Comment:
This is the dependency report that gets generated, but this won't generate using the new build structure.

Reply:
OK so how do we work around this one? Do we need to move the dependency checker logic into jacoco-test-coverage?